libbpf-tools/vfsstat.c

```diff
@@ -132,6 +132,13 @@ static void print_and_reset_stats(__u64 stats[S_MAXSTAT])
 	printf("\n");
 }
 
+static bool is_btf_enabled(void)
+{
+	if (!access("/sys/kernel/btf/vmlinux", R_OK))
```

Comment:
the problem is that /sys/kernel/btf/vmlinux is present in 5.4 kernel, while fentry/fexit is supported starting from 5.5, so on 5.4 vfsstat will be failing

We should probably implement helper to detect if fentry programs are supported and use that everywhere. I'll reply with some more details on the other PR.

Reply:
I'm sorry I forgot to check the kernel version.
I'll wait for helpers.
Thank you for your kind help!